doc/iterations/2024.3/shapes/confirmation_dialog_on_delete.adoc

```diff
@@ -0,0 +1,52 @@
+= Confirmation dialog on delete
+
+== Problem
+
+Sirius Web does not have an undo/redo mechanism.
+When an element has been deleted, either from the Explorer view, from any representation, or any other view, there is no way to retrieve it.
+All delete actions are executable in one click.
+Any accidental click on a delete action can lead to the unwanted loss of data.
+
+== Key Result
+
+No delete action should be done in one click.
+
+== Solution
+
+A confirmation dialog should be displayed when a delete action (click on the delete tool on a node, click on delete key on a node, click on the delete menu item on an object in the Explorer view) has been called.
+This dialog should avoid or drastically minimized unwanted loss of data.
+
+As a specifier, It should be possible to enable/disable this capability on every representation description (Explorer view, any diagram description...).
```

Comment:
It should be possible to be enabled/disabled by the end-user on a project.

Reply:
Ok, in place of the capability to enable/disable on every representation description, or in addition to?

If it is in addition to, I think it will lead to complex cases, as for example when the specifier disabled it on a diagram description, but the user enabled it on a project, and so who won?

Reply:
After discussion with @mbats, the solution will be in place of the capability to enable/disable on every representation description.
The end-user will not enable/disable the capability on the project, but on the dialog itself.